README.md

```diff
@@ -1,8 +1,11 @@
 # GenLayer Prototype
+
 ## Introduction
+
 Welcome to the GenLayer prototype, the first step towards a decentralized platform that combines the ease of using Python to write contracts, the access to the internet, the intelligence of the LLMs, and the security and efficiency of a blockchain.
```

Comment:
I think you should take the description form here https://docs.genlayer.com/simulator